superset-frontend/src/dashboard/components/gridComponents/Tab.jsx

```diff
@@ -53,6 +53,11 @@ const propTypes = {
   setDirectPathToChild: PropTypes.func.isRequired,
 };
 
+const StyledTab = styled.div`
+  background-color: ${({ theme }) => theme.colors.grayscale.light4};
+  padding: ${({ theme }) => theme.gridUnit * 4}px;
```

Comment:
the fix padding will break the 12 columns grid in the dashboard. please test it.